masterlist.yaml

```
@@ -20951,6 +20951,15 @@ plugins:
         util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 12
 
+  - name: 'Caranthir Tower Reborn Expanded.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/62755' ]
+    dirty:
+      # version: 1
+      - <<: *quickClean
+        crc: 0x63903DC2
+        util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 29
+  
   - name: 'Castle Volkihar Rebuilt.esp'
```

There's some trailing whitespace:

```suggestion
        itm: 29

  - name: 'Castle Volkihar Rebuilt.esp'
```